src/uct/ib/mlx5/ib_mlx5_dv.c

```diff
@@ -258,6 +269,21 @@ static ucs_status_t uct_ib_mlx5dv_md_open(struct ibv_device *ibv_device,
         goto err_free;
     }
 
+    status = ucs_mpool_init(&md->dbrec_pool, 0,
+                            sizeof(uct_ib_mlx5_dbrec_t), 0,
+                            UCS_SYS_CACHE_LINE_SIZE,
+                            ucs_get_page_size() / UCS_SYS_CACHE_LINE_SIZE - 1,
+                            UINT_MAX, &uct_ib_mlx5_dbrec_ops, "devx dbrec");
+    if (status != UCS_OK) {
+        goto err_free;
+    }
+
+    status = uct_ib_mlx5_init_mmio_devx(&md->super, &md->uar,
```

Comment:
why do we need uar in memory domain? for UMR?

Reply:
More precisely - for the cases md->pd is used in verbs: e.g. UCS_THREAD_MODE_SINGLE/SERIALIZED